ui/src/hooks/useTerminal.ts

```diff
@@ -138,9 +138,7 @@ export const useTerminal = async (el: HTMLElement, option: ICallbackOptions): Pr
             break;
           }
           case 'ZMODEM_END': {
-            if (!enableZmodem.value && zmodemStatus.value) {
-              option.i18nCallBack && message.info(option.i18nCallBack('EndFileTransfer'));
-
+            if (enableZmodem.value && zmodemStatus.value) {
               terminal?.write('\r\n');
 
               terminalStore.setTerminalConfig('zmodemStatus', false);
```

Comment:
No discrepancies have been found between the two versions of the given codes. However, if you would like to optimize them further for performance using ES6 features or improve formatting for readability, please let me know!